pkg/sql/create_schema.go

```diff
@@ -97,6 +97,7 @@ func (p *planner) createUserDefinedSchema(params runParams, n *tree.CreateSchema
 		Name:       n.Schema,
 		ID:         id,
 		Privileges: privs,
+		Version:    1,
```

Comment:
Should all of our `NewMutableCreated...Descriptor` should set the version to 1? We've (me) missed this in alot of places